From eb53e1845788ae76ef3d58034be3eb886bf60b7f Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Wed, 7 Jan 2015 10:56:58 -0500
Subject: [net] description of dma_cookie cause make xmldocs warning

Message-id: <505bbb55663c521b5bd60e42d2650444c623c6d8.1420627472.git.jbenc@redhat.com>
Patchwork-id: 102514
O-Subject: [RHEL7.1 net PATCH 5/5] net: description of dma_cookie cause make xmldocs warning
Bugzilla: 1173444
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Kyle McMartin <kmcmarti@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Myron Stowe <mstowe@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1173444
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=8460333

commit 709c48b39ecf11a81f3820c13a828c330fd832b9
Author: Masanari Iida <standby24x7@gmail.com>
Date:   Wed Oct 8 23:53:39 2014 +0900

    net: description of dma_cookie cause make xmldocs warning

    In commit 7bced397510ab569d31de4c70b39e13355046387,
    dma_cookie was removed from struct skbuff.
    But the description of dma_cookie still exist.
    So the "make xmldocs" output following warning.

    Warning(.//include/linux/skbuff.h:609): Excess struct/union
    /enum/typedef member 'dma_cookie' description in 'sk_buff'

    Remove description of dma_cookie fix the symptom.

    Signed-off-by: Masanari Iida <standby24x7@gmail.com>
    Acked-by: Dan Williams <dan.j.williams@intel.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index 2541397..265aaee 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -432,8 +432,6 @@ typedef unsigned char *sk_buff_data_t;
  * @wifi_acked_valid: wifi_acked was set
  * @wifi_acked: whether frame was acked on wifi or not
  * @no_fcs:  Request NIC to treat last 4 bytes as Ethernet FCS
- * @dma_cookie: a cookie to one of several possible DMA operations
- *  done by skb DMA functions
   * @napi_id: id of the NAPI struct this skb came from
  * @secmark: security marking
  * @mark: Generic packet mark
-- 
1.7.1